src/components/orders/AmountsDisplay/styled.tsx

```diff
@@ -0,0 +1,45 @@
+import styled from 'styled-components'
+
+import TokenImg from 'components/common/TokenImg'
+
+export const Wrapper = styled.div`
+  display: grid;
+  row-gap: 1rem;
+  justify-items: start;
+  align-items: center;
+  grid-template-columns: 10rem auto;
+  padding: 1.3rem 0;
+`
+
+export const RowTitle = styled.span`
+  display: flex;
+  align-items: center;
+  font-weight: ${({ theme }): string => theme.fontBold};
+
+  &::before {
+    content: '▶';
+    margin-right: 0.5rem;
+    color: ${({ theme }): string => theme.icon};
+    font-size: 0.75rem;
+  }
+`
+
+export const RowContents = styled.span`
+  display: flex;
+  align-items: center;
+
+  & > span:first-child {
+    margin: 0 0.5rem 0 0;
+  }
+`
+
+export const UsdAmount = styled.span`
+  color: ${({ theme }): string => theme.textPrimary1};
```

Comment:
@alfetopito What you think of using the default color (we could strictly even get rid of the `color` property on this UsdAmount). In this case a simple `opacity` could suffice.

Reply:
You are the design authority, I'll abide by anything you say :)
Seriously though, I have no preference.

Reply:
Always open to your feedback :) - Will address in this PR.